modules/luci-base/htdocs/cgi-bin/luci

```diff
@@ -1,4 +1,10 @@
-#!/usr/bin/lua
+#!/bin/sh
```

Comment:
@hnyman: The shell script sets the PATH and then invokes lua.   It is written as valid lua code.  To my knowledge, it is not possible to set PATH in lua.  This pull request is a portable way to do so for many/all versions of lua.